tools/c7n_azure/c7n_azure/resources/key_vault.py

```diff
@@ -13,15 +13,19 @@
 # limitations under the License.
```

Comment:
Please add tests for the new resource, new filter & update docs.

Reply:
Will be included in next commit.

Reply:
Please update the docs, this is a nice feature. =)